### PR TITLE
Mention `unit-test` in MIR opt test README

### PR DESCRIPTION
**Pull request by JakobDegen**
r? @RalfJung

### CODE REVIEW

File: src/test/mir-opt/README.md [hunk 1]
```diff
@@ -14,6 +14,18 @@ presence of pointers in constants or other bit width dependent things. In that c
 
 to your test, causing separate files to be generated for 32bit and 64bit systems.
 
+## Unit testing
+
+If you are only testing the behavior of your pass on some specific input (as is usually the case),
```
Comment (RalfJung) [inline src/test/mir-opt/README.md:19]:
```suggestion
If you are only testing the behavior of a particular mir-opt pass on some specific input (as is usually the case),
```

**Comment by RalfJung** [comment]
r=me with the nit fixed.

**Comment by JakobDegen** [comment]
@RalfJung fixed

**Pull request merged**